packages/donate-button/src/components/Donations/DonateButton/index.tsx

```diff
@@ -66,6 +66,20 @@ const DonateButton = ({
 	const formText = monthlyDonation ? lang.monthly : lang.oneTime;
 	const donationAmount = donationsContextValue?.donationAmount;
 
+	useEffect(() => {
```

Comment:
you can actually do this by making the actual donate form be a form, and then the button have `type="submit"`; then the browser will kick in and do all the accessibility magic to make the expected stuff just work out of the box. will make a separate ticket for that.